### ISSUE TITLE
Missing install_windows_package matcher

### ISSUE THREAD

**Issue by codylewandowski** [sethvargo/chefspec#612]
Using the windows_package resource we are getting the below errors when running spec tests. 

```
Chef::Exceptions::NoSuchResourceType
------------------------------------
Cannot find a resource for windows_package on chefspec version 0.6.1
```
Doing some research I can see that this same issue was addressed in issue https://github.com/sethvargo/chefspec/issues/372 but at the time windows_package was only included in the Windows cookbook.  It appears that windows_package was added as a Chef resource shortly after https://tickets.opscode.com/browse/CHEF-5087?page=com.atlassian.jira.plugin.system.issuetabpanels:changehistory-tabpanel. 

Searching through this repo, I don't see any results for windows_package.  Should this be added or is it still out of scope?

If it helps, I'm using Chef 12.4.1 and Chefspec 4.3.0.

**Comment by codylewandowski** [comment]
Thanks for adding this @sethvargo!!

**Comment by jedugas** [comment]
Issue's closed, but I'm seeing this error in ChefDK 0.10.0 (i.e. chef-client 12.5.1, chefspec 4.4.0) when my recipe includes the 'env' resource:

    env 'FOO' do
      value 'bar'
    end

--> "Cannot find a resource for env on chefspec version 0.6.1"

Tried adding platform/version opts to the SoloRunner constructor, but that didn't seem to matter. Specs for the recipe's remaining resources (service, cookbook_file, remote_file, template) all pass if I comment out the 'env' resource block.

**Comment by brunorubin** [comment]
Same issue here using the ```windows_package```  resource:
```
Chef::Exceptions::NoSuchResourceType
------------------------------------
Cannot find a resource for windows_package on chefspec version 0.6.1

...

  1:  # Download and installs the package
  2>> windows_package 'AWS Command Line Interface' do
  3:    source    node['awscli']['installer_url']
  4:    checksum  node['awscli']['installer_checksum']
  5:    version   node['awscli']['version']
  6:    action    :install
  7:  end  8:
```

chef-dk Version 0.10.0
chef-client Version: 12.5.1
chefspec Version: 4.4.0
